docs/modules/ROOT/pages/guides/guides-sagemaker-training.adoc

```diff
@@ -0,0 +1,329 @@
+
+[source, python]
+----
+import numpy as np
+import mlflow.keras
+from sklearn.metrics import accuracy_score, f1_score
+----
+
+NOTE: You will need to include the `scikit-learn` and `numpy` packages within:
+`<project-name>/<project-name>-pipelines/<machine-learning-pipeline-name-dashed>/<sagemaker-step-name-dashed>/pyproject.toml`
+
+You then need to modify the `test_model` function to predict on the test dataset and print out the metrics. Your
+function should resemble:
+
+[source, python]
+----
+def test_model(model, test_data):
+    """Evaluate model against test data
+    Args:
+        test_data: loaded test dataset
+    Returns:
+        resulting metrics
+    """
+    y_pred = model.predict(test_data.drop("species", axis=1))
+
+    test_acc = accuracy_score(test_data["species"], np.argmax(y_pred, axis=1))
+    test_f1 = f1_score(test_data["species"], np.argmax(y_pred, axis=1))
+
+    print(f"test_accuracy={test_acc};", flush=True)
+    print(f"test_f1={test_f1};", flush=True)
+----
+
+==== Save Model
+To save the model, we'll utilize the MLflow Model wrapper. This is not explicitly required, but the benefits of this
+approach are detailed in the Notes/Updates column of
+xref:machine-learning-pipeline-details.adoc#_sagemaker_training[Table 3]. The `mlflow` package is already imported
+within the provide training template.
+
+In the `save_model` function, we'll use the the `mlflow.keras` flavor to save out the model to the required location:
+
+[source, python]
+----
+
+def save_model(model, output_dir):
+    """Save model artifacts to disk
+    Args:
+        model: trained model
+        output_dir: directory in which to save model artifacts
+    """
+    mlflow.keras.save_model(model, output_dir)
+----
+
+=== Utilizing SageMaker Training API Service
+This section will cover how to use the SageMaker Training API service to start a SageMaker Training Job. Please see
+xref:machine-learning-pipeline-details.adoc#_sagemaker_model_training_api[SageMaker Model Training API] for all
+available routes.
+
+Before you can access the API Service, there are 2 areas where AWS Credentials will need to be updated:
+
+* `<project-name>/<project-name>-deploy/src/main/resources/apps/model-training-api-SageMaker/values.yaml`
+* `<project-name>/<project-name>-deploy/src/main/resources/apps/mlflow-ui/values.yaml`.
+
+After the credentials are added, you are ready to deploy your project. 
+You will need to run the command `mvn clean install` for your image to be pushed to ECR. To deploy the local components
+of your project, run the command `tilt up`.
+
+==== Submitting a SageMaker Training Job
+To submit a SageMaker Training Job, you will need to set the parameters of the POST request body. The required
+parameters can be found
+under the `POST /sagemaker-training-jobs` from the 
+xref:machine-learning-pipeline-details.adoc#_sagemaker_model_training_api[SageMaker Model Training API] documentation. 
+
+Using a cURL request, your request should resemble:
```

Comment:
Q: Is the odd capitalization intentional? If not, then this line as well as 2 others below should be updated.

Reply:
Some people like it this way.  I'm not one of them :), but I decided to leave it as it was.